pkg/helm/helm.go

```diff
@@ -299,6 +308,13 @@ func (r *Impl) GetConfig(kubeNamespace string, log action.DebugLog) (*action.Con
 	config := &genericclioptions.ConfigFlags{
 		Namespace: &kubeNamespace,
 	}
+	if r.discoveryBurst != -1 {
+		config.WithDiscoveryBurst(r.discoveryBurst)
+	}
+	if r.discoveryQPS != -1 {
```

Comment:
I think it's better to refer to positive values only: **> 0**

Reply:
as above

Reply:
Still, it is a floating point variable here, using ">" is safer.

Reply:
I don't agree with you, there is the positivity check in the function.

Reply:
Never mind, you are not doing any calculations to be worried about precision.